src/FSharpLint.Core/Application/Lint.fs

```diff
@@ -214,18 +216,18 @@ module Lint =
                 enabledRules.AstNodeRules |> Array.map (fun rule -> rule.Name)
             |] |> Array.concat |> Set.ofArray
 
-        let suppressionInfo = Suppression.parseSuppressionInfo allRuleNames lines
+        let suppressionInfo = Suppression.parseSuppressionInfo allRuleNames (Array.toList lines)
```

Comment:
I'd be tempted to have `parseSuppressionInfo` operate on the array to avoid creating that list with it being quite large for big files